### ISSUE TITLE
Python Tile Flipping Game

### ISSUE THREAD

**Issue by Milindcode** [sanskarmalkhede/basic-programs#58]
This is a python based Game. The objective of the game is to flip all the tiles on the board. Tiles will flip only if you flip 2 same tiles in consecutive chances.

The code involves use of functions and tkinter library of python.

Please assign this issue to me with hacktober fest label.

![image](https://user-images.githubusercontent.com/91339883/196681339-f2700efb-b328-436b-8d08-b84fa6cbd2fb.png)

**Comment by sanskarmalkhede** [comment]
Yes please go ahead and make a PR. Really looking forward to your contribution.
Make sure that you have starred the repo.

**Comment by Milindcode** [comment]
@sanskarmalkhede I think you have closed the issue accidently. Please open it and assign it to me

**Comment by sanskarmalkhede** [comment]
Is it ok now?

**Comment by Milindcode** [comment]
@sanskarmalkhede Yes. Thank you .
